rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -90,7 +90,7 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
                       max_decoded_width * planes,
                       max_decoded_height,
                       tjpf,
-                      TJFLAG_FASTDCT, &x1_diff, &crop_width_diff,
+                      TJFLAG_ACCURATEDCT, &x1_diff, &crop_width_diff,
```

Comment:
Are we not converging with TJFLAG_FASTDCT. Has that been checked?
Also was there any performance impact because of change from fast to accurate?

Reply:
Will be triggering a training to confirm convergence with FastDCT and use FastDCT if its converging. With regards to performance, I've not observed any performance drop with AccurateDCT algorithm